src/components/MenuItem.css

```diff
@@ -5,3 +5,14 @@
   margin: 1.25rem;
   text-align: left;
 }
+
+.menu__buttons {
+  border: none;
+  background-color: white;
+  padding: 0 0.5rem 0 0.5rem;
```

Comment:
this can be summarized to `padding: 0 .5rem`. I'd recommend using `em` for padding though